README.md

```diff
@@ -1,3 +1,164 @@
+3. From the left menu select **Library** and activate: 
+    - Google Drive API
+    - Google Docs API
+    - Google Slides API
+    - Google Sheets API
+    - Google Drive Activity API
+4. Go back to **APIs & Services** menu.
+5. Create a new OAuth consent screen. Select the option **OAuth consent screen**  on the menu bar. If you would like to limit your application to organization-only users, select **Internal**, otherwise, select **External** option, then, fill the form with the data you would use for your project.
+6. Go back to **APIs & Services** menu.
+7. Create a new Client. Select the option **Credentials**, and on the menu bar, select **Create credentials**, a dropdown menu will be displayed, then, select **OAuth Client ID** option. 
+8. Then, a select input will ask the type of Application type that will be used, select **Web application**, then, fill the form, and on **Authorized redirect URIs** introduce this URI:
+    https://<your_mattermost_instance>/plugins/com.mattermost.apps/apps/google-drive/oauth2/remote/complete
+9. After the Client has been configured, on the main page of **Credentials**, on the submenu **OAuth 2.0 Client IDs** will be displayed the new Client and the info can be accessible whenever you need it.
+
+## Configuration
+
+After [installing](#installation)) the app and [creating a project](#create-a-google-cloud-project)):
+1. As a Mattermost system admin user, run the ``/drive configure`` command.
+2. In the configuration modal, enter your Client ID, Client Secret and Service Account.
+    - Client ID and Client Secret: Used to generate the link to let the user connect with their Google Drive account.
+    - Service Account: Is the way the app will get the credentials to manage webhook notifications
+
+
+# Admin guide
+
+## Slash commands
+- ``/drive configure:``: This command will enable the connect and disconnect commands which allow the user to link with their Google Drive account. You're prompted to provide a Client ID and Client Secret.
+
+
+# End user guide
+
+## Get started
+
+## Use ``/drive`` commands
+
+- ``/drive help``: This command will show all current commands available for this application.
+- ``/drive connect``: This command will create a new link, and clicking on that link redirects the user to select the Google account that will be used to execute all the available actions.
+- ``/drive create [docs | slide | sheets]``: This command will display a new modal where the data will be asked. 
+    - Title: Name of the file to be created.
+    - Message: Optional. Applicable only if a file is shared in the channel.
+    - File Access: Choose to share with members in the channel, with anyone who has the link, or choose to keep it private.
+- ``/drive notifications [start | stop]``: This command will start or stop notification. After a user runs the ``/drive connect`` command, notifications will be posted to the private channel with the user.
```

Comment:
Was there a feature requested to subscribe a Google doc's updates to a channel?

Reply:
Actually, what `/drive notification start` does is to subscribe to any update, any comment, inside the Google Drive of the person who executed the command. The notifications are received in the channel between the Bot and the user

Reply:
Right, these are "DM notifications", which now that I think about it should be made clear here. It says "notifications will be posted to the private channel with the user.", when it should say "notifications will be posted to bot's DM channel with the user."

For the comment above, I wasn't meaning to ask about the current functionality. I was asking if there was a separate requested feature not yet implemented, to subscribe a Google doc's updates to a channel.

Reply:
The README change was made in commit [7e4380d](https://github.com/mattermost/mattermost-app-google-drive/pull/2/commits/7e4380d3b8a7c5488325798adbaf5134988eb129), but as I far as I'm concern, a subscription to a given Google Doc wasn't one of the requirements (at least for this release).